pilot/pkg/model/config_test.go

```diff
@@ -22,10 +22,10 @@ import (
 
 	"github.com/davecgh/go-spew/spew"
 
+	mccpb "istio.io/api/mixer/v1/config/client"
 	networking "istio.io/api/networking/v1alpha3"
 	"istio.io/istio/pilot/pkg/config/memory"
 	"istio.io/istio/pilot/pkg/model"
-	mccpb "istio.io/istio/pilot/pkg/networking/plugin/mixer/client"
```

Comment:
if we are removing this should we completely remove this code? It should be dead code once we switch everything over to the api/mixer/... variant?

Reply:
Agreed.. but planning to cherry-pick this to release-1.6.. so removal of code should be just in 1.7 pr?

Reply:
Also, this is used by mixer.go so cant remove it completely yet...